src/js/view/components/LogViewerList.js

```diff
@@ -85,3 +85,7 @@ const LogViewerList = props => {
 };
 
 export default LogViewerList;
+
+const isNotOnlyWhitespace = str => {
```

Comment:
This looks great now, the rows are matching in height like they should. Nice fix with the unicode. I learned something new there.  
This might be only me being picky, but I would like to see this function above the component. Under memoizeProps perhaps? It feels better somehow to have the last line of the component be the export statement.

Reply:
You're right, it should be moved, my bad!